tests/test_baseclass.py

```diff
@@ -431,12 +431,40 @@ def load_prediction_data(self, *args, **kwargs):
 
 
 class TestScoreEstimator:
-    def test_score_estimator_fails_if_no_train_test_data_available(self, iris_dataset):
+    def test_score_estimator_if_no_train_test_data_available(
```

Comment:
What is this test doing? Hard to understand from the function name

Reply:
This test checks if the `score_estimator ` correctly calculates the score estimator without any test data has been created
```suggestion
    def test_score_estimator_creates_train_test_data(
```